main/postfix/APKBUILD

```diff
@@ -164,6 +164,18 @@ package() {
 		_mvline "^\s*$map" "$map" dynamicmaps.cf
 	done
 	rm makedefs.out
+
+	# fix /etc/postfix/postfix-files so that "postfix set-permissions" can run
+	sed -i \
+		-e "/shlib_directory\/postfix-/d" \
+		-e "/meta_directory\/makedefs.out/d" \
+		-e "/manpage_directory/d" \
+		-e "/config_directory\/LICENSE/d" \
+		-e "/config_directory\/TLS_LICENSE/d" \
+		-e "/config_directory\/.\+\.cf\.default/d" \
+		-e "/config_directory\/main.cf/d" \
+		-e "/config_directory\/master.cf/d" \
```

Comment:
Why are main and master configuration files removed?

Reply:
I don't find a read-only mount used with some Docker containers to be a compelling reason for removing these two files. Particularly when as you said, postfix is going to fail when they are missing.